### PR TITLE
Lesson 7 HTML Render

### PR DESCRIPTION
**Pull request by ganttArt**

### CODE REVIEW

File: students/christopher_gantt/lesson07/html_render.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+    def __init__(self, content=None, **kwargs):
+        if content is None:
+            self.contents = []
+        else:
+            self.contents = [content]
+        self.attributes = kwargs
+
+
+    def append(self, new_content):
```
Comment (natasha-aleksandrova) [inline students/christopher_gantt/lesson07/html_render.py]: class methods should only have one line space (module functions have two lines space)

File: students/christopher_gantt/lesson07/html_render.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python3
+
+"""
+A class-based system for rendering html.
+"""
+
+class Element(object):
```
Comment (natasha-aleksandrova) [inline students/christopher_gantt/lesson07/html_render.py]: no need to extend from object in python3

File: students/christopher_gantt/lesson07/html_render.py [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+    def _open_tag(self):
+        open_tag = ["<{}".format(self.tag)]
+        for key, value in self.attributes.items():
+            open_tag.append(' {}="{}"'.format(key, value))
+        return "".join(open_tag)
+
+
+    def render(self, out_file, cur_ind=""):
+        if self.tag == "html":
+            out_file.write('<!DOCTYPE html>\n')
```
Comment (natasha-aleksandrova) [inline students/christopher_gantt/lesson07/html_render.py]: it's best to keep this as part of inheritance structure - `Html` class would have its own `render` method and then call parent's `render()` to do execute the rest of the code.

File: students/christopher_gantt/lesson07/html_render.py (hunk 2 above)
Comment (ganttArt) [reply students/christopher_gantt/lesson07/html_render.py]: To clarify, this means that I can write the class as...   class Element():    ...and not add the object?

Comment (natasha-aleksandrova) [reply students/christopher_gantt/lesson07/html_render.py]: `class Element:` is python3 style

**Pull request merged**